docs/pages/blog/mui-x-v5.md

```diff
@@ -0,0 +1,191 @@
+- We renamed most selectors to have a consistent naming convention, making it easier to deduce their name or infer purpose.
+- We restructured our state so that each feature has a single substate, and is the only one to update it (`state.filter` is only managed by our `useGridFilter` hooks which exposes methods for both internal and 3rd party code to interact with this state).
+
+The work on that topic is far from over. We have several developments in progress or under discussion to improve the developer experience when using the advanced features of the grid.
+Here are a few that should be release in the following months:
+
+- Strict typing of event listeners and publishers.
+- Examples for the event listeners.
+- Documentation and examples for selectors.
+- Add the ability to export and restore some parts of the grid state.
+
+#### Synchronous state initialization
+
+In previous versions, the state was first populated with default values, and then in a `useEffect`, given the values provided as props (`props.pageSize` for instance), or derived from the props (the sorted and filtered rows derived from the `props.rows`, `props.sortModel` and `props.filterModel`).
+This was causing an additional render with useless data, and the we had to be careful to avoid flickering between the fake and real data.
+In v5, the state is initialized synchronously during the first render.
+
+Note that for now, the state updates coming from controlled props are still asynchronous.
+If you pass `props.pageSize`, we will apply it to the state in a `useEffect`, and therefore if you read the state just after the render (for instance in a `useLayoutEffect`), you will still see the old version.
+
+## Reduced style specificity for easier customization
+
+In previous versions most of the various `DataGrid` and `DataGridPro` components had a CSS specificity of 2, meaning that style overrides and customizations were harder, requiring you to look at the DOM tree in order to pick the correct selector.
+With MUI X v5 we have reduced the CSS specificity of most of the internal `DataGrid` and `DataGridPro` components to 1.
+This will enable developers to more easily change the look and feel of the grid's components.
+
+**Before**
+
+```jsx
+const useStyles = makeStyles(() => ({
+  root: {
+    '& .MuiDataGrid-toolbarContainer': {
+      padding: 50,
+    },
+  },
+}));
+
+const MyCustomToolbar = () => {
+  return <GridToolbarContainer>My custom toolbar</GridToolbarContainer>;
+};
+
+export default function App() {
+  const classes = useStyles();
+
+  return (
+    <div style={{ height: 400, width: '100%' }} className={classes.root}>
+      <DataGrid
+        components={{
+          Toolbar: MyCustomToolbar,
+        }}
+      />
+    </div>
+  );
+}
+```
+
+**After**
+
+```jsx
+const useStyles = makeStyles(() => ({
```

Comment:
`makeStyles`'s API is deprecated, could we use the `styled()` API instead?

Also, people can now used the sx `prop`

Reply:
Updated